<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="6" hostname="YP1009099DT" name="com.practice.tests.AnnotationTransformerTest" tests="8" failures="2" timestamp="17 Apr 2019 08:40:48 GMT" time="0.093" errors="0">
  <testcase name="test1" time="0.016" classname="com.practice.tests.AnnotationTransformerTest">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at com.practice.tests.AnnotationTransformerTest.test1(AnnotationTransformerTest.java:10)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="test2" time="0.000" classname="com.practice.tests.AnnotationTransformerTest">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test1" time="0.000" classname="com.practice.tests.AnnotationTransformerTest">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase name="test2" time="0.000" classname="com.practice.tests.AnnotationTransformerTest">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.031" classname="com.practice.tests.AnnotationTransformerTest">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at com.practice.tests.AnnotationTransformerTest.test2(AnnotationTransformerTest.java:15)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test1" time="0.000" classname="com.practice.tests.AnnotationTransformerTest">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase name="test1" time="0.031" classname="com.practice.tests.AnnotationTransformerTest">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase name="test2" time="0.015" classname="com.practice.tests.AnnotationTransformerTest">
    <skipped/>
  </testcase> <!-- test2 -->
</testsuite> <!-- com.practice.tests.AnnotationTransformerTest -->
